src/main/java/mekanism/api/IMekWrench.java

```diff
@@ -13,7 +13,7 @@
 	/* easier compat with BC */
 	default boolean canUseWrench(EntityPlayer player, EnumHand hand, ItemStack stack, RayTraceResult rayTrace)
 	{
-		return canUseWrench(stack, player, rayTrace.getBlockPos() != null ? rayTrace.getBlockPos() : BlockPos.ORIGIN);
+		return canUseWrench(stack, player, rayTrace.getBlockPos());
```

Comment:
So what happens if rayTrack.getBlockPos is null?

Reply:
From what intellij was telling me it is never null. This is because for the rayTrace to have a null blockpos it would have to be a RayTraceResult for looking at entities. And the ray trace result is always made for the wrench from onBlockActivated, type methods.